nrf_rpc/CHANGELOG.rst

```diff
@@ -19,6 +19,12 @@ Added
 
 * Added the :kconfig:option:`CONFIG_NRF_RPC_ZCBOR_BACKUPS` Kconfig option for setting up the number of zcbor backups.
 
+Changes
+=======
+
+* Added a possibility for transport in each group to initialize independently of other groups.
+* This enables the RPC to work for those groups which are ready, even though initialization of others may have failed.
```

Comment:
```suggestion
* Enabled independent initialization of transport in each group.
  This makes RPC available for the groups that are ready, even though initialization of others may have failed.
```
Not separate bullets, because this is very much cause and effect.

Reply:
Right.